tests/unit/test_notebooks_python.py

```diff
@@ -38,3 +38,9 @@ def test_baseline_deep_dive_runs(notebooks):
 def test_surprise_deep_dive_runs(notebooks):
     notebook_path = notebooks["surprise_svd_deep_dive"]
     pm.execute_notebook(notebook_path, OUTPUT_NOTEBOOK, kernel_name=KERNEL_NAME)
+
+
+@pytest.mark.notebooks
```

Comment:
so I guess vw is working because it is preinstalled on the dsvm, right?

We should think of adding a installer. Not sure if this PR is the best moment though

Reply:
yeah, i'm close to resolving the issue of installing the python bindings on conda so that would be a preferable way to do this, they way it is being done on the dsvm is just through apt-get install vowpalwabbit